Packages/Status/Sources/Status/Row/StatusRowViewModel.swift

```diff
@@ -62,6 +67,14 @@ public class StatusRowViewModel: ObservableObject {
 
     isFiltered = filter != nil
   }
+    
+  func markSeen() {
```

Comment:
What about also having the settings check here? So it's not adding anything to the cache if the feature is off?